http/http.go

```diff
@@ -138,9 +158,7 @@ func (s *Server) newResponse(r *http.Request) (Response, error) {
 		ASN:        autonomousSystemNumber,
 		ASNOrg:     asn.AutonomousSystemOrganization,
 		Hostname:   hostname,
-		UserAgent:  userAgent,
 	}
-	s.cache.Set(ip, response)
```

Comment:
Why isn't this cached?

Reply:
it could be, but I left that to the caller

Reply:
oh. i see. You don't cached the useragent info. I'll switch this around.